Code/Framework/AzFramework/AzFramework/Physics/Common/PhysicsTypes.h

```diff
@@ -24,11 +24,12 @@ namespace Physics
 
 namespace AzPhysics
 {
-    //! Default Scene Names and Crc32
-    static constexpr const char* DefaultPhysicsSceneName = "DefaultScene";
-    static constexpr const AZ::Crc32 DefaultPhysicsSceneId = AZ_CRC_CE(DefaultPhysicsSceneName);
-    static constexpr const char* EditorPhysicsSceneName = "EditorScene";
-    static constexpr const AZ::Crc32 EditorPhysicsSceneId = AZ_CRC_CE(EditorPhysicsSceneName);
+    //! Maximum number of scenes, Default Scene Names and Crc32
```

Comment:
I think this doxygen markup will only be applied to the first line below. If you wrap the lines in //! @{ and //! @} I think the markup will apply to all 5 lines.

Reply:
Ahh, you're right. Not worth another AR though. I'll do the change in the future.